druntime/test/importc_compare/Makefile

```diff
@@ -1,5 +1,13 @@
 TESTS := importc_compare
 
+# FIXME: fails on Alpine v3.21 with conflicting struct declarations in the C headers:
+# /usr/include/asm-generic/fcntl.h(195): Error: struct `importc_includes.flock` conflicts with struct `importc_includes.flock` at /usr/include/fcntl.h(24)
+ifeq ($(OS),linux)
+    ifeq (1,$(shell which apk &>/dev/null && echo 1))
+        TESTS :=
```

Comment:
so that here you can use `ifneq ($(IS_ALPINE),1)`

Reply:
Different file and druntime test dir, plus I don't know if this is Alpine-specific, possibly even v3.21 specific. (This test wasn't in v2.110, so no results with LDC for this test yet.)